src/test/java/net/comments/sample/TrainingRoomUnlimitedTest.java

```diff
@@ -0,0 +1,56 @@
+package net.comments.sample;
+
+import org.hamcrest.MatcherAssert;
+import org.hamcrest.collection.IsIterableContainingInAnyOrder;
+import org.hamcrest.collection.IsIterableContainingInOrder;
+import org.testng.annotations.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class TrainingRoomUnlimitedTest {
+
+    @Test
+    public void testGetEnteredPersonExactOrder() {
+
+        List<Person> enteredPerson = new ArrayList<Person>();
+        AttentiveStudent student1 = new AttentiveStudent("Student1");
+        AttentiveStudent student2 = new AttentiveStudent("Student2");
+        BadStudent student3 = new BadStudent("Student3");
+        BadStudent student4 = new BadStudent("Student4");
+        MultiTeacher teacher1 = new MultiTeacher("Teacher1");
+        MultiTeacher teacher2 = new MultiTeacher("Teacher2");
+
+        enteredPerson.add(student1);
+        enteredPerson.add(student2);
+        enteredPerson.add(student3);
+        enteredPerson.add(student4);
+        enteredPerson.add(teacher1);
+        enteredPerson.add(teacher2);
+
+        MatcherAssert.assertThat(enteredPerson, IsIterableContainingInOrder.contains(student1, student2, student3, student4, teacher1, teacher2));
+
+    }
+
+    @Test
+    public void testGetEnteredPersonAnyOrder() {
```

Comment:
I don't see where `TrainingRoomUnlimited` is used. What do you want to test with this unit test?

Reply:
@extsoft Yesterday, I had some doubts about it. I don't know why I check that some objects is present in list without any order. I think it's useless test. Am I right?

Reply:
The current implementation is useless. But if you start using `TrainingRoomUnlimited`, it'll become useful.